File: projects/java_calculator/src/test/java/com/example/CalculatorTest.java

package com.example;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class CalculatorTest {

    @Test
    public void testAdd() {
        Calculator calculator = new Calculator();
        assertEquals(3, calculator.add(1, 2));
        assertEquals(5, calculator.add(2, 3));
        assertEquals(100, calculator.add(50, 50));
    }
}

File: projects/kotlin_calculator/src/test/kotlin/com/example/CalculatorTest.kt
package com.example

import org.junit.Assert.assertEquals
import org.junit.Test

class CalculatorTest {

    @Test
    fun testAdd() {
        val calculator = Calculator()
        assertEquals(5, calculator.add(2, 3))
        assertEquals(0, calculator.add(1, -1))
        assertEquals(-1, calculator.add(1, -2))
        assertEquals(100, calculator.add(50, 50))
    }
}